### ISSUE TITLE
Missing alternative text on images

### ISSUE THREAD

**Issue by tysongach** [thoughtbot/constable#687]
Some of images, mainly avatars, are missing alternative text, which poses accessibility issues.

You can read about the accessibility implications of images with missing alt text, here: https://dequeuniversity.com/rules/axe/3.2/image-alt

Also, in some cases in the app, our alt text for an avatar is something like “Picture of Tyson Gach”. This is a bit redundant because most (if not all) screen readers annouce and image element as an image, so it would say “image, picture of Tyson Gach” - we can probably remove the “picture of” part.

It seems like announcement _show_ pages are fine, but the _index_ page is missing them:

<img width="1160" alt="Screen Shot 2019-08-28 at 15 03 58" src="https://user-images.githubusercontent.com/903327/63885104-9e0dc480-c9a5-11e9-8c6f-8448da324253.png">
